integration-tests/testsuite/src/it/java/org/graalvm/tests/integration/utils/BuildAndRunCmds.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020, Red Hat Inc. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.graalvm.tests.integration.utils;
+
+/**
+ * BuildAndRunCmds
+ *
+ * The last command is used to run the final binary.
+ * All previous commands are used to build it.
+ *
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public enum BuildAndRunCmds {
+    // Note that at least 2 command are expected. One or more to build. The last one to run the app.
+    QUARKUS_FULL_MICROPROFILE(new String[][]{
+            new String[]{"mvn", "clean", "compile", "package", "-Pnative"},
+            new String[]{Commands.isThisWindows ? "target\\quarkus-runner" : "./target/quarkus-runner"}
+    }),
+    MICRONAUT_HELLOWORLD(new String[][]{
+            new String[]{"mvn", "package"},
+            new String[]{"native-image", "-jar", "target/helloworld.jar", "target/helloWorld"},
```

Comment:
I think that we should also be able to control which `native-image` to use.

Reply:
@zakkak I see. That is not the case in this implementation, it just picks what you've got on your PATH.   I wanted it to start simple...

Reply:
Same as in https://github.com/graalvm/mandrel/pull/83#discussion_r459993937